lib/navigation/route_names.dart

```diff
@@ -0,0 +1,6 @@
+const String waiting_for_game_to_start = '/waiting_for_game_to_start';
```

Comment:
Following up on the "pollute the namespace" conversation, you could consider putting routeName constants inside the relevant classes, eg. 

```Dart
class CreateGameScreen2 extends StatelessWidget {
  static const routeName = '/create_game2';
...
```

and then in the MaterialApp routes:

```Dart
CreateGameScreen2.routeName: (context) => CreateGameScreen2(),
```

Reply:
beautiful